README.md

```diff
@@ -100,6 +48,47 @@ mutant --include lib --require virtus --use rspec Virtus::Attribute.build
 mutant --include lib --require virtus --use rspec Virtus::Attribute#type
 ```
 
+Configuration
+-------------
+
+Occasionally mutant will produce a mutation with an infinite runtime.  When this happens
+mutant will look like it is running indefinitely without killing a remaining mutation.  To
+avoid mutations like this, consider adding a timeout around your tests.  For example, in
+RSpec you can add the following to your `spec_helper`:
+
+```ruby
+config.around(:each) do |example|
+  Timeout.timeout(5_000, &example)
+end
+```
+
+which will fail specs which run for longer than 5 seconds.
+
+Rails
+-------
+
+Assuming you are using rspec, you can mutation test Rails models by adding the following lines to your Gemfile:
+
+```ruby
```

Comment:
Can we move this gemfile addition outside of the `Rails` context? Its not rails specific.